frontend_server/test/server_test.dart

```diff
@@ -273,38 +274,6 @@ Future<int> main() async {
       streamController.close();
     });
 
-    test('accept', () async {
-      final StreamController<List<int>> inputStreamController =
-        new StreamController<List<int>>();
-      final ReceivePort acceptCalled = new ReceivePort();
-      when(compiler.acceptLastDelta()).thenAnswer((Invocation invocation) {
-        acceptCalled.sendPort.send(true);
-      });
-      final int exitcode = await starter(args, compiler: compiler,
-        input: inputStreamController.stream,
-      );
-      expect(exitcode, equals(0));
-      inputStreamController.add('accept\n'.codeUnits);
-      await acceptCalled.first;
-      inputStreamController.close();
-    });
-
-    test('reject', () async {
-      final StreamController<List<int>> inputStreamController =
-        new StreamController<List<int>>();
-      final ReceivePort rejectCalled = new ReceivePort();
-      when(compiler.rejectLastDelta()).thenAnswer((Invocation invocation) {
-        rejectCalled.sendPort.send(true);
-      });
-      final int exitcode = await starter(args, compiler: compiler,
-        input: inputStreamController.stream,
-      );
-      expect(exitcode, equals(0));
-      inputStreamController.add('reject\n'.codeUnits);
-      await rejectCalled.first;
-      inputStreamController.close();
-    });
-
```

Comment:
I guess if the methods are made nops then this test code doesn't have to be deleted either.